core/arch/arm/plat-versal2/conf.mk

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2023-2024, Advanced Micro Devices, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-2-Clause
```

Comment:
We don't normally add Copyright and SPDX comments in the .mk files. I don't see any harm in adding them if you want them, but in that case, please put the SPDX-License-Identifier in the first line as we do in all source files.